faster_raster.go

```diff
@@ -374,9 +372,6 @@ func (r *Rasterizer) calculateScaleForDocument(pageCount int) {
 //  will be used. Width overrides any scale factor and will be rendered to as close
 //  to that exact dimension as possible, if it's supplied.
 func (r *Rasterizer) processOne(req *RasterRequest) {
-	bounds := new(C.fz_rect)
```

Comment:
Were we before and are we now still managing the allocation/free cycle properly around bounds?

Reply:
They decided to pass it around as a stack variable instead since managing pointers is such a nightmare. The bounds are an instance of `fz_rect`, which [contains only 2 floats](https://github.com/ArtifexSoftware/mupdf/blob/7f77313e6d60fc917146b80cce2f9c1ab0152349/include/mupdf/fitz/geometry.h#L155-L156), so I guess the performance impact is negligible.

Reply:
PS: I believe the allocation of `bounds` was safe before, because the memory was allocated in the Go heap via `new` and we weren't keeping a reference to it in C (I hope) :)